java/test/src/main/java/org/ray/api/test/ActorReconstructionTest.java

```diff
@@ -47,6 +47,8 @@ public int getPid() {
   @Test
   public void testActorReconstruction() throws InterruptedException, IOException {
     TestUtils.skipTestUnderSingleProcess();
+    // By design. No lineage cache when direct actor call is enabled.
+    TestUtils.skipTestIfDirectActorCallEnabled();
```

Comment:
We can add another actor reconstruction test for direct-call actors to verify that the actor can be reconstructed, but no tasks are replayed.

Reply:
I removed this skip and changed one of the asserts to `Assert.assertEquals(value, options.isDirectCall ? 1 : 4);`